libraries/bezier-rs/src/bezier/solvers.rs

```diff
@@ -386,6 +390,15 @@ impl Bezier {
 		.flat_map(|bezier| self.intersections(bezier, None, None))
 		.collect()
 	}
+
+	/// Returns a cubic bezier which joins this and the provided bezier curves.
+	/// The resulting path formed by the Bezier curves is smooth up to the first derivative.
```

Comment:
Not super sure if "smooth" is the best descriptor here but I don't feel too strongly about it.

Reply:
I'll update the comment to say 'continuous' instead.